clusterloader2/README.md

```diff
@@ -1,21 +1,23 @@
 # ClusterLoader
 
-## Running ClusterLoader
+ClusterLoader2 (CL2) is a "bring your own yaml" Kubernetes load testing tool
+being an official K8s scalability and performance testing framework.
 
-To run ClusterLoader type:
-```
-go run cmd/clusterloader.go --kubeconfig=kubeConfig.yaml --testconfig=config.yaml --provider=local
-```
-OR
-```
-./run-e2e.sh --testconfig=config.yaml --provider=local
-```
+The CL2 tests are written in yaml using the semi-declarative paradigm.
+A test defines a set of states in which a cluster should be
+(e.g. I want to run 10k pods, 2k cluster-ip services, 5 daemon-sets, etc.)
+and CL2 sequentially transitions the cluster through these states.
```

Comment:
and specifies how fast (e.g. pod throughput) a given state should be reached.
In addition, it defines which performance characteristics should be measured.

Reply:
Done.